invenio_app_ils/records_relations/api.py

```diff
@@ -424,6 +455,96 @@ def remove(self, first, second, relation_type):
         RecordRelationsMetadata.remove_metadata_from(
             second, relation_type.name, first.pid.pid_value, first._pid_type
         )
-
-        # return the allegedly modified record
         return first, second
+
+
+class RecordRelationsSequence(RecordRelations):
+    """Add/Remove operations for Sequence relations."""
+
+    allowed_metadata = ["title"]
```

Comment:
you should remove this. This is a check for "extra" metadata of the relation, e.g. the `note` for the sibling relation of type `other`. It is not for the metadata of the document/series.

Reply:
Thanks for the clarification.